src/web/mjs/engine/BookmarkManager.mjs

```diff
@@ -211,4 +212,22 @@ export default class BookmarkManager extends EventTarget {
     compareBookmarks( a, b ) {
         return a.title.manga.toLowerCase() < b.title.manga.toLowerCase() ? -1 : 1 ;
     }
+
+    /**
+     * Check newer content within bookmarks
+     */
+    async checkNewContent()
+    {
+        const checkManga = async function checkManga(bookmark){
+            //Retrieve connector
+            let mangaconnector = Engine.Connectors.find( connector => {
+                return connector.id === bookmark.key.connector ;
+            } );
+            //Prepare manga
+            let manga = new Manga(mangaconnector,bookmark.key.manga,bookmark.title.manga);
+            await manga.checkNewContent();
+        };
+        //Check all bookmarks for new content
+        await Promise.all(this.bookmarks.map(checkManga));
```

Comment:
Spawning concurrent updates for all bookmarks at the same time may strain resources and has a high risk to getting IP banned by some websites

Reply:
Yes, but i've planned to use a throttling system within the connector. Will not request for merge as long as it's not done (got much work to do)